src/_assets/css/_variables.scss

```diff
@@ -12,7 +12,9 @@ $site-color-nav-links: #6E7274;
 $site-color-body: #4A4A4A;
 $site-color-body-light: rgba(74/255, 74/255, 74/255, .8);
 $site-color-footer: #303c42;
+$site-color-paragraph: #555;
 $site-color-primary: $flutter-color-blue-500;
+$site-color-title: #212C34;
```

Comment:
It looks like we're using this color only in the showcase page. Let's move it to the _showcase.scss file.